### PR TITLE
Implement WalletService

### PR DESCRIPTION
**Pull request by Liosh-a**

### CODE REVIEW

File: BobManager.API/BobManager.API/Controllers/Api/WalletController.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
```
Comment (ander21) [inline BobManager.API/BobManager.API/Controllers/Api/WalletController.cs]: Are you realy use all of this links?

File: BobManager.API/BobManager.API/Controllers/Api/WalletController.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+        public WalletController(IWalletService _walletService)
+        {
+            walletService = _walletService;
+        }
+
+        [HttpGet]
+        public Task<CollectionResultDto<SpendingDto>> GetSpendings()
+        {
+
```
Comment (ander21) [inline BobManager.API/BobManager.API/Controllers/Api/WalletController.cs]: Remove empty strings, please

File: BobManager.API/BobManager.API/Startup.cs [hunk 3]
```diff
@@ -42,15 +48,24 @@ public Startup(IConfiguration configuration)
```
Comment (ander21) [inline BobManager.API/BobManager.API/Startup.cs:41]: Remove empty space

File: BobManager.API/BobManager.Domain/Services/Implementation/WalletService.cs [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+using BobManager.DataAccess.Entities;
+using BobManager.DataAccess.Interfaces;
+using BobManager.Domain.Services.Abstraction;
+using BobManager.Dto.DtoModels;
+using BobManager.Dto.DtoResults;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```
Comment (ander21) [inline BobManager.API/BobManager.Domain/Services/Implementation/WalletService.cs:8]: Are you realy use all of this links?

File: BobManager.API/BobManager.Domain/Services/Implementation/WalletService.cs [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+        {
+            repository = _repository;
+            mapper = _mapper;
+        }
+
+        public async Task<CollectionResultDto<SpendingDto>> GetSpendigs()
+        {
+            var spending = await repository.GetAllInclude(x=>x.SpendingCategory);
+
+
```
Comment (ander21) [inline BobManager.API/BobManager.Domain/Services/Implementation/WalletService.cs]: 30-31 Remove empty spaces, please

**Review by ander21 — COMMENTED** [review]
Reslove, please!

**Review by NerevitVasyan — APPROVED** [review]
Approve

**Review by Firto — APPROVED** [review]
Approve

**Review by ander21 — APPROVED** [review]
Appprove

**Pull request merged**